etc/module.xml

```diff
@@ -3,6 +3,10 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="../../../../../lib/internal/Magento/Framework/Module/etc/module.xsd">
```

Comment:
Should be: `xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd"`